PROTOCOL.md

```diff
@@ -160,6 +160,8 @@ description|`String`| User-provided description for this table
 format|[Format Struct](#Format-Specification)| Specification of the encoding for the files stored in the table
 schemaString|[Schema Struct](#Schema-Serialization-Format)| Schema of the table
 partitionColumns|`Array[String]`| An array containing the names of columns by which the data should be partitioned
+createdTime|Long| The time this metadata change was introduced, as milliseconds since the epoch
```

Comment:
createdTime|`Option[Long]`|The time when this metadata action is created, in milliseconds since the Unix epoch.